mbed_app.json

```diff
@@ -24,8 +24,7 @@
     },
     "target_overrides": {
         "*": {
-            "platform.stdio-convert-newlines": true,
-            "platform.stdio-baud-rate": 115200
```

Comment:
115200 works fine for me... I'm using the latest version of mbedhtrun (0.0.15).
I'm inclined to revert it to 115200, because our [examples.json](https://github.com/ARMmbed/mbed-os/blob/ad40b1b2672d59b058f7c104fdc612d0dddf3ac3/tools/test/examples/examples.json#L512) says so.
@jamesbeyond Could you confirm we have had no issue with baud rate in CI?